pkg/registration/hub/lease/clocksynccontroller.go

```diff
@@ -103,8 +103,9 @@ func (c *clockSyncController) sync(ctx context.Context, syncCtx factory.SyncCont
 		return err
 	}
 	// When the agent's lease get renewed, the "now" on hub should close to the RenewTime on agent.
-	// If the two time are not close(over 1 lease duration), we assume the clock is out of sync.
```

Comment:
@qiujian16  Before, we treat this condition as a warning, so the threshold is low, over 1 minute the the condition is show.

But now as we are using the condition to control agent's update actions, I think it's better to keep it the same as the duration we used in lease controller.

Reply:
do we use the leaseDurationSeconds in the spec?

Reply:
Goot point, updated.